utils/rpms/daos.spec

```diff
@@ -657,826 +657,3 @@ fi
 %doc README.md
 # No files in a shim package
 %endif
```

Comment:
Currently one can build source rpms with this file.   Is there a way to reference the external changelog?